lib/pusher/http_client.ex

```diff
@@ -11,7 +11,8 @@ defmodule Pusher.HttpClient do
   end
 
   defp process_response_body(body) do
-    unless body == "", do: body |> JSX.decode!, else: nil
+    :ok = validate_response_body(body)
```

Comment:
That's fine I'm just saying that using `JSX.decode` will say if the body was JSON or not. If you get :error then you know that the JSON decoding didn't work